govcd/external_network_v2_test.go

```diff
@@ -0,0 +1,156 @@
+// +build extnetwork network functional openapi ALL
+
+/*
+ * Copyright 2020 VMware, Inc.  All rights reserved.  Licensed under the Apache v2 License.
+ */
+
+package govcd
+
+import (
+	"fmt"
+
+	"github.com/vmware/go-vcloud-director/v2/types/v56"
+	. "gopkg.in/check.v1"
+)
+
+func (vcd *TestVCD) Test_CreateExternalNetworkV2NsxT(check *C) {
```

Comment:
missing test Test_CreateExternalNetworkV2NsxV or it's name is `Test_CreateExternalNetworkV2PortGroup` ?

Reply:
Yes. `Test_CreateExternalNetworkV2PortGroup`

Reply:
so maybe better name it Test_CreateExternalNetworkV2NsxV ?:)

Reply:
Can be. Done.